CHANGELOG.md

```diff
@@ -1,5 +1,42 @@
 # Change Log
 
+## 8.6.0 (April 722, 2019)
```

Comment:
@robertohuertasm I think you meant "April 22, 2019" and not "April 722, 2019".

Just updated the package to the latest release and spotted this error, so this is just a heads-up from a random user.

Reply:
We were in a hurry and when you are in a hurry :hankey: :hankey: happens.

Reply:
Thanks for catching that!

Reply:
Fixed